system_metrics_collector/src/moving_average_statistics/moving_average.cpp

```diff
@@ -39,7 +39,7 @@ double MovingAverageStatistics::max() const
 
 double MovingAverageStatistics::min() const
 {
-  return MovingAverageStatistics::getStatistics().min;
```

Comment:
This seems unrelated, can you move this to another PR?

Reply:
Removed from this PR.